bash/gast_to_code/bash_gast_to_code_converter.py

```diff
@@ -51,7 +51,7 @@ def handle_if(gast, lvl=0):
 
     def handle_var_assign(gast):
         value = router.gast_to_code(gast["varValue"], "bash")
-        return router.gast_to_code(gast["varId"], "bash") + "=" + value
+        return router.gast_to_code(gast["varId"], "bash") + " = " + value
```

Comment:
FYI: the reason that I did not add spaces originally is bc bash treats spaces in a strange way. Not sure what is the right way but wanted to give you a heads up!